File: exs/mundo_1/python/017.py
"""
Desafio 017

Problema: Faça um programa que leia o comprimento do cateto oposto e
          do cateto adjacente de um triângulo retângulo. Calcule e
          mostre o comprimento da hipotenusa.

Resolução do problema:
"""
from math import hypot
cateto_oposto = int(input('Informe o cateto oposto: '))
cateto_adjacente = int(input('Informe o cateto adjacente: '))

hipostenusa = hypot(cateto_oposto, cateto_adjacente)  # H² = C² + C² // H² = (C² + C²) ** (1/2)
print('Hipotenusa: {:.2f}'.format(hipostenusa))

File: exs/mundo_1/javascript/017.js
/*
Desafio 017

Problema: Faça um programa que leia o comprimento do cateto oposto (co) e 
          do cateto adjacente (ca) de um triângulo retângulo, calcule e mostre 
          o comprimento da hipotenusa

Resolução do problema:
*/

const input = require('readline-sync').question

let cateto_oposto = Number(input('Informe o cateto oposto: '))
let cateto_adjacente = Number(input('Informe o cateto adjacente: '))

let hipotenusa = Math.hypot(cateto_oposto, cateto_adjacente)

console.log(`Hipotenusa: ${hipotenusa}`);